seeds/gems/text_classification/README.md

```diff
@@ -0,0 +1,6 @@
+# Text classification with embeddings
+
+| Username      | Dataset | Notebook    |
```

Comment:
Would you please put eval results also as a column? And the format in nlpprogress.com is better - 
https://nlpprogress.com/english/sentiment_analysis.html

<h3 id="imdb" style="margin: 10px 0px; padding: 0px; border: 0px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-variant-numeric: inherit; font-variant-east-asian: inherit; font-variant-alternates: inherit; font-weight: 700; font-stretch: inherit; font-size: 24px; line-height: inherit; font-family: &quot;Lucida Grande&quot;, Calibri, Helvetica, Arial, sans-serif; font-optical-sizing: inherit; font-kerning: inherit; font-feature-settings: inherit; font-variation-settings: inherit; vertical-align: baseline; color: rgb(34, 34, 34); letter-spacing: -1px; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(242, 242, 242); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">IMDb</h3><p style="margin: 10px 0px 15px; padding: 0px; border: 0px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-variant-numeric: inherit; font-variant-east-asian: inherit; font-variant-alternates: inherit; font-weight: 400; font-stretch: inherit; font-size: 16px; line-height: inherit; font-family: &quot;Myriad Pro&quot;, Calibri, Helvetica, Arial, sans-serif; font-optical-sizing: inherit; font-kerning: inherit; font-feature-settings: inherit; font-variation-settings: inherit; vertical-align: baseline; color: rgb(55, 55, 55); letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(242, 242, 242); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">The<span> </span><a href="https://ai.stanford.edu/~ang/papers/acl11-WordVectorsSentimentAnalysis.pdf" style="margin: 0px; padding: 0px; border: 0px; font: inherit; vertical-align: baseline; text-decoration: none; color: rgb(15, 121, 208); text-shadow: none; transition: text-shadow 0.5s ease 0s;">IMDb dataset</a><span> </span>is a binary sentiment analysis dataset consisting of 50,000 reviews from the Internet Movie Database (IMDb) labeled as positive or negative. The dataset contains an even number of positive and negative reviews. Only highly polarizing reviews are considered. A negative review has a score ≤ 4 out of 10, and a positive review has a score ≥ 7 out of 10. No more than 30 reviews are included per movie. Models are evaluated based on accuracy.</p>

Model | Accuracy | Paper / Source
-- | -- | --
XLNet (Yang et al., 2019) | 96.21 | XLNet: Generalized Autoregressive Pretraining for Language Understanding
BERT_large+ITPT (Sun et al., 2019) | 95.79 | How to Fine-Tune BERT for Text Classification?
BERT_base+ITPT (Sun et al., 2019) | 95.63 | How to Fine-Tune BERT for Text Classification?
ULMFiT (Howard and Ruder, 2018) | 95.4 | Universal Language Model Fine-tuning for Text Classification
Block-sparse LSTM (Gray et al., 2017) | 94.99 | GPU Kernels for Block-Sparse Weights
oh-LSTM (Johnson and Zhang, 2016) | 94.1 | Supervised and Semi-Supervised Text Categorization using LSTM for Region Embeddings
Virtual adversarial training (Miyato et al., 2016) | 94.1 | Adversarial Training Methods for Semi-Supervised Text Classification
BCN+Char+CoVe (McCann et al., 2017) | 91.8 | Learned in Translation: Contextualized Word Vectors

<br class="Apple-interchange-newline">IMDb
The [IMDb dataset](https://ai.stanford.edu/~ang/papers/acl11-WordVectorsSentimentAnalysis.pdf) is a binary sentiment analysis dataset consisting of 50,000 reviews from the Internet Movie Database (IMDb) labeled as positive or negative. The dataset contains an even number of positive and negative reviews. Only highly polarizing reviews are considered. A negative review has a score ≤ 4 out of 10, and a positive review has a score ≥ 7 out of 10. No more than 30 reviews are included per movie. Models are evaluated based on accuracy.

Model	Accuracy	Paper / Source
XLNet (Yang et al., 2019)	96.21	[XLNet: Generalized Autoregressive Pretraining for Language Understanding](https://arxiv.org/pdf/1906.08237.pdf)
BERT_large+ITPT (Sun et al., 2019)	95.79	[How to Fine-Tune BERT for Text Classification?](https://arxiv.org/pdf/1905.05583.pdf)
BERT_base+ITPT (Sun et al., 2019)	95.63	[How to Fine-Tune BERT for Text Classification?](https://arxiv.org/pdf/1905.05583.pdf)
ULMFiT (Howard and Ruder, 2018)	95.4	[Universal Language Model Fine-tuning for Text Classification](https://arxiv.org/abs/1801.06146)
Block-sparse LSTM (Gray et al., 2017)	94.99	[GPU Kernels for Block-Sparse Weights](https://s3-us-west-2.amazonaws.com/openai-assets/blocksparse/blocksparsepaper.pdf)
oh-LSTM (Johnson and Zhang, 2016)	94.1	[Supervised and Semi-Supervised Text Categorization using LSTM for Region Embeddings](https://arxiv.org/abs/1602.02373)
Virtual adversarial training (Miyato et al., 2016)	94.1	[Adversarial Training Methods for Semi-Supervised Text Classification](https://arxiv.org/abs/1605.07725)
BCN+Char+CoVe (McCann et al., 2017)	91.8	[Learned in Translation: Contextualized Word Vectors](https://arxiv.org/abs/1708.00107)

Reply:
Hi @sidgoogle , just did some revisions. I added a dataset description and the accuracy. 

As for the table header, should I include columns for paper/source? And the model name? I wasn't sure.